<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaceShips</class>
 <widget class="QWidget" name="PlaceShips">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="select_groupBox">
         <property name="title">
          <string>Select Ship:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="ship1_radioButton">
            <property name="text">
             <string>1x1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ship2_radioButton">
            <property name="text">
             <string>1x2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ship3_radioButton">
            <property name="text">
             <string>1x3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ship4_radioButton">
            <property name="text">
             <string>1x4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ship5_radioButton">
            <property name="text">
             <string>1x5</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QComboBox" name="col_comboBox"/>
            </item>
            <item>
             <widget class="QComboBox" name="row_comboBox"/>
            </item>
            <item>
             <widget class="QPushButton" name="place_pushButton">
              <property name="text">
               <string>Place</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="rotate_pushButton">
            <property name="text">
             <string>Rotate</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="place_groupBox">
         <property name="title">
          <string>Place Ship</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="Grid" name="board_Grid" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="done_pushButton">
       <property name="text">
        <string>DONE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Grid</class>
   <extends>QWidget</extends>
   <header>grid.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
